packages/storybook/src/stories/ino-switch/ino-switch.stories.js

```diff
@@ -66,7 +66,6 @@ export const DefaultUsage = () => /*html*/ `
    <ino-switch
      id="custom-switch"
      class="customizable-switch"
-     ino-color-scheme="${select('ino-color-scheme', INO_COLOR_SCHEMES, 'primary')}"
```

Comment:
Regarding the removal of the `ino-color-scheme` property: Setting the color scheme will have no effect because it gets overwritten by the customizable CSS variables